invenio_oauthclient/contrib/settings.py

```diff
@@ -59,6 +67,7 @@ def __init__(
             precedence_mask=precedence_mask,
             signup_options=signup_options,
             logout_url=logout_url,
+            hide=LocalProxy(lambda: self.hide_when()),
```

Comment:
```suggestion
            hide=LocalProxy(lambda: self.hide_when() if callable(self.hide_when) else self.hide_when),
```
Supporting simple values next to callables could simplify the configuration a bit and drop the requirement of the `CONDITION_FALSE` import/definition above.
It would, of course, require an extra line of documentation.